components/layout/SideNav.tsx

```diff
@@ -341,11 +343,6 @@ const MobileWrap = styled.div`
   @media (${breakUp("lg")}) {
     display: none;
   }
-
-  /* Hides MobileWrap contents when Sidebar is toggled and a Section is active. Not tabbable when not in view. */
-  &.visuallyhidden {
-    visibility: hidden;
-  }
```

Comment:
Because there is now only one "SlidePane", there is no longer a duplicate MobileWrap that needs to be hidden